package.json

```diff
@@ -41,25 +41,25 @@
     "grunt-shell-spawn": "^0.3.2",
     "jasmine-core": "^2.2.0",
     "jsdom": ">= 0.1.23 < 4.0.0",
-    "karma": "0.12.32",
-    "karma-chrome-launcher": "^0.1.8",
-    "karma-coffee-preprocessor": "^0.2.1",
-    "karma-coverage": "^0.3.1",
+    "karma": "0.13.14",
```

Comment:
Looks like it is fine. Took `2.413 secs / 2.658 secs` fast for me. The only thing that is slow, is travis is a fresh install of binaries of npm and bower.